Source/GSXibLoader.m

```diff
@@ -742,6 +757,17 @@ - (void) dealloc
   [super dealloc];
 }
 
+- (NSString*) description
+{
+  NSMutableString *description = [[super description] mutableCopy];
```

Comment:
Same as before, no need for a `mutableCopy` when a `stringWithFormat` will do just nicely